sgrep_lint/tests/run-lint-tests.sh

```diff
@@ -147,6 +147,7 @@ $SGREP --strict --config=tests/python/good.yaml tests/lint
 #echo TODO: disabled sgrep-rules regression testing for now
 rm -rf /tmp/sgrep-rules && git clone https://github.com/returntocorp/sgrep-rules /tmp/sgrep-rules
 $SGREP --dangerously-allow-arbitrary-code-execution-from-rules --strict --test --test-ignore-todo /tmp/sgrep-rules
```

Comment:
Why not just replace both these lines with `cd /tmp/sgrep-rules && make test`?

Reply:
This runs locally using python while sgrep-rules make test needs a binary executable called sgrep-lint

Reply:
Should/couldn't we just alias the local python run to $sgrep-lint?

Reply:
I tried (not very hard) but couldn't get make to pickup the new alias